### ISSUE TITLE
[a11y][iOS] Map markers are not visible to VoiceOver

### ISSUE THREAD

**Issue by computerjazz** [react-native-community/react-native-maps#3114]
# Bug

When using VoiceOver on iOS, tapping on a map marker has no effect. 

## Environment info

React native info output:

```bash
 React Native Environment Info:
    System:
      OS: macOS 10.14.6
      CPU: x64 Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz
      Memory: 57.57 MB / 8.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 12.9.1 - ~/.nvm/versions/node/v12.9.1/bin/node
      Yarn: 1.17.3 - ~/.yarn/bin/yarn
      npm: 6.10.2 - ~/.nvm/versions/node/v12.9.1/bin/npm
      Watchman: 4.7.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.2, macOS 10.14, tvOS 12.2, watchOS 5.2
      Android SDK:
        Build Tools: 28.0.3
        API Levels: 28
    IDEs:
      Android Studio: 3.3 AI-182.5107.16.33.5264788
      Xcode: 10.2.1/10E1001 - /usr/bin/xcodebuild
    npmPackages:
      @types/react: ^16.7.18 => 16.7.18 
      @types/react-native: ^0.57.27 => 0.57.27 
      react: 16.8.6 => 16.8.6 
      react-native: 0.59.8 => 0.59.8 
    npmGlobalPackages:
      react-native-cli: 2.0.1
      react-native-create-library: 3.1.2
      react-native-git-upgrade: 0.2.7
      react-native-rename: 2.4.1
```

Library version: 0.25.0

## Steps To Reproduce

1. Create a map with a custom marker
2. Turn on VoiceOver
3. Attempt to tap on marker
4. Nothing happens

Describe what you expected to happen:

1. Black outline appears around marker
2. VoiceOver reads accessibility label of marker
3. If marker is tappable, it may be double-tapped to trigger callout, etc

**Comment by rborn** [comment]
#3115 merged, closing
